src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ title, user, body, comments }) => (
```

Comment:
If you pass `post` as one instance it'd be better to work with types and make better props structure
```suggestion
export const PostInfo = ({ post}) => {
  const { title, user, body, comments } = post;
  
  return (
```